TypeScriptReact.sublime-settings

```diff
@@ -0,0 +1,4 @@
+{
+    "auto_complete_triggers" : [ {"selector": "source.tsx", "characters": "."} ],
```

Comment:
Is there a reason this isn't just added to the existing sublime-settings file? (Not that it matters much).